pkg/pusher/pusher.go

```diff
@@ -79,9 +83,16 @@ func (s *Service) chunksWorker() {
 			chunksInBatch++
 			s.metrics.TotalChunksToBeSentCounter.Inc()
 
+			t, err := s.tag.GetByAddress(ch.Address())
+			if err != nil {
+				s.logger.Errorf("pusher: error while getting tag: %v", err)
```

Comment:
This log line should be of Debug severity as it exposes the error.
```suggestion
				s.logger.Debugf("pusher: get tag by address %s: %v", ch.Address(), err)
```